src/store/index.ts

```diff
@@ -42,23 +42,30 @@ import { SafeReducerMap } from 'src/logic/safe/store/reducer/types/safe'
 import { LS_NAMESPACE, LS_SEPARATOR } from 'src/utils/constants'
 
 const CURRENCY_KEY = `${CURRENCY_REDUCER_ID}.selectedCurrency`
-export const LS_CONFIG: RLSOptions | LoadOptions = {
+
+export const SAVE_LS_CONFIG: RLSOptions = {
   states: [ADDRESS_BOOK_REDUCER_ID, CURRENCY_KEY, APPEARANCE_REDUCER_ID, CONFIG_REDUCER_ID],
   namespace: LS_NAMESPACE,
   namespaceSeparator: LS_SEPARATOR,
   disableWarnings: true,
+}
+
+export const LOAD_LS_CONFIG: LoadOptions = {
+  states: [ADDRESS_BOOK_REDUCER_ID, CURRENCY_KEY, APPEARANCE_REDUCER_ID],
+  namespace: LS_NAMESPACE,
+  namespaceSeparator: LS_SEPARATOR,
+  disableWarnings: true,
   preloadedState: {
```

Comment:
Why did you remove the preloaded state?

Reply:
Because it was overwriting my initial chain id with the last chain id.

Reply:
Can you not dispatch the initialised id after?

Reply:
I've done that now. We already had a function for this, it just wasn't called in the right place.